lib/hydra/derivatives/shell_based_processor.rb

```diff
@@ -66,14 +68,43 @@ def execute_with_timeout(timeout, command, context)
         end
 
         def execute_without_timeout(command, context)
+          exit_status = nil
+          err_str = ''
           stdin, stdout, stderr, wait_thr = popen3(command)
           context[:pid] = wait_thr[:pid]
           stdin.close
-          out = stdout.read
           stdout.close
-          err = stderr.read
-          stderr.close
-          raise "Unable to execute command \"#{command}\". Exit code: #{wait_thr.value}\nError message: #{err}" unless wait_thr.value.success?
+          files = [stderr]
```

Comment:
we weren't doing anything with the output of stdout, so there's really no reason to read it.  I closed it on the preceding line.